llvm/include/llvm/Object/COFF.h

```diff
@@ -861,6 +896,8 @@ class COFFObjectFile : public ObjectFile {
   // Either coff_load_configuration32 or coff_load_configuration64.
   const void *LoadConfig = nullptr;
   const chpe_metadata *CHPEMetadata = nullptr;
+  const coff_dynamic_reloc_table *DynamicRelocTable = nullptr;
+  std::shared_ptr<WritableMemoryBuffer> HybridBufferRef;
```

Comment:
This seems like a leftover? A `std::shared_ptr` in these classes feels like a very unusual thing (we don't have any occurrances of that before), so that would require a bit of motivation - but it seems to be unused in this patch still?

Reply:
Yes, it's a leftover from experiments that I dropped, sorry about that.